src/components/viewLayout/__tests__/viewLayoutNavGroup.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { shallowComponent } from '../../../../config/jest.setupTests';
+jest.mock('../../../routes', () => {
```

Comment:
As far as I understand, this is canonical way of mocking a module - you have to create your mock **before** it is imported. Here, we want to test `ViewLayout`, which imports `routes` directly, so we need to mock routes first.

All that to test the path of menu with navigation hierarchy. We never use navigation hierarchy and I seriously doubt we ever will, so maybe we should just remove the support for it - we can always bring it back from git, if needed. But I already removed another module and decided that this time I will try to test it.